subsys/zigbee/common/zigbee_helpers.c

```diff
@@ -839,4 +845,75 @@ void user_input_indicate(void)
 		}
 	}
 }
+
+/* Function to enable sleepy behavior for End Device. */
+void zigbee_configure_sleepy_behavior(bool enable)
+{
+	if (enable) {
+		zb_set_rx_on_when_idle(ZB_FALSE);
+		LOG_INF("Enabled sleepy end device behavior.");
+	}
+	else {
+		zb_set_rx_on_when_idle(ZB_TRUE);
+		LOG_INF("Disabling sleepy end device behavior.");
+	}
+}
+#endif
+
+/**@bried Calculate bottom addresss of RAM bank with given id.
+ *
+ * @param[in] bank_id  ID of RAM bank to get start address of.
+ *
+ * @return    Start address of RAM bank.
+ */
+static inline u32_t ram_bank_bottom_addr(u8_t bank_id)
+{
+	u32_t bank_bottom_addr = RAM_START_ADDRESS + \
+		bank_id * RAM_BANK_0_7_SECTION_SIZE * RAM_BANK_0_7_SECTIONS_NBR;
+	return bank_bottom_addr;
+}
+
+void zigbee_power_down_unused_ram(void)
+{
+	/* ID of top RAM bank. Depends of amount of RAM available at SoC. */
+#if defined CONFIG_BOARD_NRF52840DK_NRF52840 || \
+	defined CONFIG_BOARD_NRF52833DK_NRF52833
+	u8_t     bank_id                 = 8;
 #endif
```

Comment:
I'd put `#else` and a proper error here or a warning and return.
The code below uses `bank_id` variable, so if you leave it as is it will lead to non-intuitive compile time errors.

Reply:
Done